### PR TITLE
485 Leverage dockerignores to better sanitize slim down our images

### PR DESCRIPTION
**Pull request by kacperpONS**
# Has the DDL schema changed?
*Pull requests which include schema changes should be labelled with the `schema change` label for visibility*

*The [common entity model pom.xml](ssdc-rm-common-entity-model/pom.xml) file `project.version` must be bumped
appropriately if there are any common entity code changes*

* [ ] This PR includes DDL schema changes and has been labelled correctly, bumping to new schema
  version: 
* [ ] The POM has been updated with an appropriate version bump if required

# Motivation and Context

Added the .dockerignore files

# What has changed

# How to test?

# Links

https://trello.com/c/JvVs8wuU/485-leverage-docker-ignores-to-better-sanitize-slim-down-our-images-5

# Screenshots (if appropriate):

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,2 +1,10 @@
-Makefile
-**/*.example
+# Ignore everything...
+*
+# ... exept for:
+!groundzero_ddl/
+!patches/
+!ui-persmissions/
```
Comment (ryangrundy7) [inline .dockerignore:6]: This is spelt wrong

File: .dockerignore [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+!build_groundzero_dll.sh
```
Comment (ryangrundy7) [inline .dockerignore:7]: Same with this one

File: .dockerignore (hunk 1 above)
Comment (ryangrundy7) [inline .dockerignore:6]:
```suggestion
!ui-permissions/
```

**Review by ryangrundy7 — APPROVED** [review]

File: .dockerignore [hunk 3]
```diff
@@ -1,2 +1,10 @@
-Makefile
-**/*.example
+# Ignore everything...
+*
+# ... exept for:
```
Comment (AdamHawtin) [inline .dockerignore:3]:
```suggestion
# ... except for:
```

**Pull request merged**